### PR TITLE
#1 Finalize Project

### PR DESCRIPTION
**Pull request by Amtresu**
## In this repo I've finished creating a project centered around the Spacex Launches api endpoint

- Selected an API that contains numeric data
- Displayed the data in a list
- List can be filtered by name
- User can click on the arrow and be taken to a more detailed page
- Used React documentation, components, props, Router
- Used Redux,
- https://www.loom.com/share/6238e537b5d14bdeb4763db086a408bb
image

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            <header>
+                <div className="header-nav">
+                    <div>
+                        <img src={arrow} alt='back-arrow' className="arrow icon" />
+                    </div>
+                    <div>
+                        <img src={mic} alt="mic" className="mic icon" />
+                        <img src={settings} alt="settings icon" className="settings" />
+                    </div>
+                </div>
```
Comment (1GORDON) [inline src/components/Header.js:22]: Great job @Amtresu 🤝

- [ ] Kindly make the clickable icons of your Navbar visible enough. At the moment they are not visible enough. I almost failed to see the back button. I would recommend a white color just like the one provided in the design.
navbar

File: README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
```
Comment (1GORDON) [inline README.md:53]: Great job @Amtresu 🤝

- [ ] Kindly make sure the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Amtresu ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [x] Great video presentation, however, you never highlighted a piece of code that you were proud or excited about. I can see you highlighted a search feature in your UI instead of code. Maybe you could highlight the piece of code you wrote to build this search feature.

_Check the comments under the review._

## Optional suggestions

- [x] You may always write your commit messages with an imperative tense something like `Update Package.json` instead of `Updating Package.json`

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+Original design is from Nelson Sakwa on Behance.
```
Comment (omar-muhamad) [inline README.md]: - [x] As mentioned by the previous reviewer, Please add the links to Nelson Sakwa and the design Behance link.
You can do it like that:
I give credits to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)).

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @Amtresu,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Gitflow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [x] As mentioned by the previous reviewer, Please make your comments messages in the imperative tense.
For example the last 2 commit message suppose to be: 
`implement suggested changes`
`fix linter errors`

image


- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @Amtresu,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Gitflow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**